### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by crash2bash**
- [DEMO LINK](https://crash2bash.github.io/layout_html-form/) <br>
- [TEST REPORT LINK](https://crash2bash.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,7 +9,91 @@
...
+        <label for="yes">Yes</label>
+        <input type="radio" name="cats" id="no" value="No">
+        <label for="no">No</label>
+      </div>
+      <div class="form__block-wrapper">
+        <label for="color">What is your favorite color?</label>
+        <input type="color" name="color" id="color">
+      </div>
+      <div class="form__block-wrapper">
+        <label for="sleep">What time you going to the bed? </label>
```
Comment (earthhuman41) [inline src/index.html]: you can do smthng like this
```
<label>
  <input>
</label>
```
and you can't to provide "for" attribute

File: src/index.html [hunk 2]
```diff
@@ -9,7 +9,91 @@
...
+    <fieldset class="form__personal-facts form__block">
+      <legend>An interesting fact about you!</legend>
+      <div class="form__block-wrapper">
+        <span>Do you love cats? </span>
+        <input type="radio" name="cats" id="yes" value="Yes">
+        <label for="yes">Yes</label>
+        <input type="radio" name="cats" id="no" value="No">
+        <label for="no">No</label>
+      </div>
+      <div class="form__block-wrapper">
```
Comment (earthhuman41) [inline src/index.html]: you dont need div here

File: src/index.html (hunk 1 above)
Comment (crash2bash) [reply src/index.html]: Готово

File: src/index.html (hunk 2 above)
Comment (crash2bash) [reply src/index.html]: All done

**Pull request closed without merging**